packages/f-services/tests/globalisation.test.js

```diff
@@ -0,0 +1,109 @@
+import { getLocale, getTheme } from '../src/globalisation';
+
+describe('Globalisation services', () => {
+    describe('getLocale', () => {
+        it('returns "en-GB" if tenantConfigs does NOT contain the locale from tenantString', () => {
+            // Arrange
+            const tenantConfigs = {
+                'da-DK': 'dk'
+            };
+            const tenantString = 'it-IT';
+            const globalTenant = {
+                locale: 'da-DK'
+            };
+
+            // Act
+            const result = getLocale(tenantConfigs, tenantString, globalTenant);
+
+            // Assert
+            expect(result).toBe('en-GB');
+        });
+
+        it('returns "en-GB" if tenantConfigs does NOT contain the locale from tenantString or globalTenant', () => {
+            // Arrange
+            const tenantConfigs = {
+                'da-DK': 'dk'
+            };
+            const tenantString = '';
+            const globalTenant = {
+                locale: 'it-IT'
+            };
+
+            // Act
+            const result = getLocale(tenantConfigs, tenantString, globalTenant);
+
+            // Assert
+            expect(result).toBe('en-GB');
+        });
+
+        it('returns the correct locale if tenantConfigs does contain the locale from tenantString', () => {
```

Comment:
I understand the wording of the description here in that it matches up with the rest of the `it()`s in this `describe()`, but it scans a bit weirdly to read. I'd suggest 
```suggestion
        it('returns the correct locale if tenantConfigs contains the locale from tenantString', () => {
```